index.html

```diff
@@ -15,18 +15,35 @@
         <h1 class='logo'>Ski Meow</h1>
       </div>
       <div>
-        <p id='intro-text'>
+        <p id='introText'>
           Ski Meow helps you choose the ski resort with the most ideal
           conditions for a great powder day! Ski Meow reports epic snowfall,
           and if the wind and chill will be worth it. Don't wait for the snow
           to melt, let's Ski Meow!
         </p>
       </div>
+      <div id='loginForm'>
+        <div class='input-container'>
+          <label for='username'>Username</label>
+          <input type='text' class='inputField' name='username' placeholder='Enter username'>
```

Comment:
Good choice to shorten your placeholder given the amount of space available.

Reply:
Thanks, not all the words were visible before.